src/Data/Semigroup/Cancellative.hs

```diff
@@ -198,8 +189,6 @@ instance SumCancellative a => RightCancellative (Sum a)
 
 -- Product instances
 
-instance Num a => Commutative (Product a)
```

Comment:
https://github.com/obsidiansystems/commutative-semigroups/pull/2 would replace. Please me know what you think about that PR, and whether it will make something that is OK for for this library!